test/Feersum.Tests/SpecTests.fs

```diff
@@ -196,25 +197,26 @@ let ``spec tests parse result`` s =
 let ``Test new lexer`` s =
     task {
         let! sourceText = File.ReadAllTextAsync(Path.Join(specDir, s))
-        let lexer = Lexer(sourceText, "test.scm")
+
+        let lexer =
+            (Lex.tokenise sourceText "test.scm")
+                .GetEnumerator()
 
         use timeout = new CancellationTokenSource(TimeSpan.FromSeconds(2.0))
```

Comment:
Probs don’t need the token here. Just a bail on count to check for endless loops.